docs/DeadmanAlert.md

```diff
@@ -0,0 +1,56 @@
+### Deadman Alert
+
+The `deadman` function is used in the context of a task to alert when a group stops reporting data.
+For example, say you have the following task definition:
+
+```
+option task = {
+    ...
+    period: 1h,
+    every:  15m,
+    ...
+}
+from(bucket: "telegraf/autogen")
+    |> range(start: -period)
+    |> filter(fn: (r) => r._measurement == "cpu")
+```
+
+In order to be alerted when the above task stops reporting data for a group, you would define a deadman alert using the `deadman` function like so:
+```
+option task = {
+    ...
+    period: 1h,
```

Comment:
Tasks don't have periods? What is this meant to be?

Reply:
My mistake, I'll remove. It's just the range of the query.